### PR TITLE
New package: ircii-20141122

### PR DESCRIPTION
**Pull request by diogoleal**

### CODE REVIEW

**Comment by diogoleal** [comment]
I added ```only_for_archs="i686 x86_64"```because I could not compile is package for ARM

**Comment by xtraeme** [comment]
This is not the solution if compilation is broken

File: srcpkgs/ircii/template [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Template file for 'ircii'
+pkgname=ircii
+version=20141122
+revision=1
+wrksrc=${pkgname}-${version}
+build_style=gnu-configure
+only_for_archs="i686 x86_64"
+configure_args="--prefix=/usr --mandir=/usr/share/man"
```
Comment (xtraeme) [inline srcpkgs/ircii/template]: This configure_args line is redundant, those are set by default.

File: srcpkgs/ircii/template [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Template file for 'ircii'
+pkgname=ircii
+version=20141122
+revision=1
+wrksrc=${pkgname}-${version}
```
Comment (xtraeme) [inline srcpkgs/ircii/template]: This wrksrc line is redundant, wrksrc is set to ${pkgname}-${version} by default.

File: srcpkgs/ircii/template [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+build_style=gnu-configure
+only_for_archs="i686 x86_64"
+configure_args="--prefix=/usr --mandir=/usr/share/man"
+hostmakedepends="ncurses-devel  libressl-openssl"
+makedepends="ncurses-devel libressl-devel"
+depends="lftp ncurses"
```
Comment (xtraeme) [inline srcpkgs/ircii/template]: This depends line is incorrect, why does it depend on lftp? and depending on ncurses is also probably wrong.

File: srcpkgs/ircii/template [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+short_desc="The iircII is an IRC and ICB client"
```
Comment (xtraeme) [inline srcpkgs/ircii/template]: `the iircII is` is redundant.

File: srcpkgs/ircii/template [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+maintainer="Diogo Leal <diogo@diogoleal.com>"
+license="BSD"
```
Comment (xtraeme) [inline srcpkgs/ircii/template]: If the license is BSD you should install the license file with `vlicense`.

File: srcpkgs/ircii/template [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+homepage="http://www.eterna.com.au/ircii/"
+distfiles="http://ircii.warped.com/ircii-20141122.tar.gz"
```
Comment (xtraeme) [inline srcpkgs/ircii/template]: replace `20141122` with `${version}`

File: srcpkgs/ircii/template (hunk 3 above)
Comment (beefcurtains) [inline srcpkgs/ircii/template]: Make it ``virtual?smtp-server``.

Comment (beefcurtains) [inline srcpkgs/ircii/template]: Actually, that would be a ``hostmakedepend``, unless you don't ship the ``ircbug`` script. Not sure if you want it as a plain ``depends``, it's only used by said script to send bug reports.

File: srcpkgs/ircii/template (hunk 4 above)
Comment (beefcurtains) [inline srcpkgs/ircii/template]: Starting ``short_desc`` with "A(n)" is useless, too. Everything is a(n) something.

File: srcpkgs/ircii/template (hunk 3 above)
Comment (beefcurtains) [inline srcpkgs/ircii/template]: It'll actually compile in support for checking email in ``/var/spool/mail`` when it's present at compile time. Having the ``hostmakedepend`` will enable this feature.

File: srcpkgs/ircii/template [hunk 7]
```diff
@@ -0,0 +1,17 @@
+# Template file for 'ircii'
+pkgname=ircii
+version=20141122
+revision=1
+wrksrc=${pkgname}-${version}
+build_style=gnu-configure
+only_for_archs="i686 x86_64"
```
Comment (beefcurtains) [inline srcpkgs/ircii/template]: It should be possible to cross compile this by [patching out most of the tests](http://sprunge.us/DVRC), running ``autoconf`` (in ``pre_configure``), and then adding the [missing defines](http://sprunge.us/aiZF) to ``defs.h`` (in ``post_configure``). Every test that tries to compile and run a program should be patched out.

**Comment by xtraeme** [comment]
This package is not installing the files into DESTDIR correctly.

**Comment by diogoleal** [comment]
I will fix it and solve the problem of cross-compiller

**Comment by xtraeme** [comment]
Stalled for 3 months... feel free to resubmit it later.

**Pull request closed without merging**